server/src/main/java/io/druid/client/cache/HybridCacheProvider.java

```diff
@@ -24,7 +24,7 @@
 import com.google.common.base.Preconditions;
 import com.google.inject.name.Named;
 
-public class HybridCacheProvider implements CacheProvider
+public class HybridCacheProvider extends HybridCacheConfig implements CacheProvider
```

Comment:
is it possible to get HybridCacheConfig instance get injected instead of extending same ?

Reply:
I tried initially to do this, but it's not obvious how to inject sub-providers in this case. Other cache providers also extend config

Reply:
its alright if not possible or too difficult.